app/src/main/java/com/geeksville/mesh/ui/UsersFragment.kt

```diff
@@ -37,56 +43,6 @@ class UsersFragment : ScreenFragment("Users"), Logging {
 
     private val model: UIViewModel by activityViewModels()
 
-    private fun popup(node: NodeEntity) {
-        if (!model.isConnected()) return
```

Comment:
we're missing a similar check on the refactored code. except for "more details" the other actions only work while connected.